model/src/test/kotlin/config/RepositoryConfigurationTest.kt

```diff
@@ -90,6 +90,25 @@ class RepositoryConfigurationTest : WordSpec({
                   - id: "vulnerability id"
                     reason: "INEFFECTIVE_VULNERABILITY"
                     comment: "vulnerability comment"
+                configurations:
+                  package_configurations:
```

Comment:
I'm not completely happy with the naming here, but wanted to reuse `package_configurations` as they are a known term within ORT.
Any suggestions for a better naming?

Reply:
I'd personally either go for `package_configurations` as single top level entry, or
```
configurations:
  packages:
```
Tendency to the former. 
What's the reason for nesting it in `Configurations`? Consistency?

Reply:
> What's the reason for nesting it in `Configurations`? Consistency?

Yes. That is the only reason. I'll change it to `package_configurations` as a single top level entry.

Reply:
Let's get @oss-review-toolkit/core-devs input on that first.

Reply:
Already changed it, as I didn't see your answer. However: I'd be fine with changing it back after some input from the other @oss-review-toolkit/core-devs.

Reply:
I also prefer a single top level entry `package_configurations`. Similarly `curation` could be added as another top level entry later.

Reply:
> Similarly `curation` could be added as another top level entry later.

I didn't get this bit. Do you mean refactoring as follows ?

```
curations:
  license_findings:
  packages:
```
to
```
package_curations:
license_finding_curations:
```

Reply:
> I also prefer a single top level entry package_configurations. Similarly curation could be added as another top level entry later.

Curations were already added in this PR: https://github.com/oss-review-toolkit/ort/pull/4381
in this form:
```yml
curations:
  packages:
  - id: "Maven:id..."
    curations:
    ....
```